pyphare/pyphare/pharesee/hierarchy/hierarchy.py

```
@@ -556,11 +556,44 @@ def plot2d(self, **kwargs):
 
         return fig, ax
 
+    def plot3d(self, **kwargs):
+        """!HAX!"""
+        time = kwargs.get("time", self._default_time())
+        usr_lvls = kwargs.get("levels", self.levelNbrs(time))
+        default_qty = None
+        if len(self.quantities()) == 1:
+            default_qty = self.quantities()[0]
+        qty = kwargs.get("qty", default_qty)
+        for lvl_nbr, lvl in self.levels(time).items():
```

Ruff (0.8.2)

Loop control variable `lvl` not used within loop body

Rename unused `lvl` to `_lvl`

(B007)